src/PullRequest/OpenPrUseCase.php

```diff
@@ -16,6 +17,7 @@
     public function __construct(
         private EntityManagerInterface $entityManager,
         private SlackMessageRepositoryInterface $slackMessageRepository,
+        private GitHubSlackMappingRepositoryInterface $gitHubSlackMappingRepository,
```

Comment:
Probably you don't need to have EM outside of repositories, if you implement a save method like:

```php
    public function save(GitHubSlackMapping $mapping): void
    {
        $this->_em->persist($mapping);
        $this->_em->flush();
    }
```

Reply:
Thank you @Chemaclass for your suggestions! Feel free to contribute too!